### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mvitz**
Updates all dependencies and uses the latest complete-spring-mvc version with Graal.

### CODE REVIEW

**Comment by mvitz** [comment]
@joyheron now that complate-spring-mvc 0.7.0 is released this should be merged.

File: package.json [hunk 1]
```diff
@@ -1,12 +1,18 @@
 {
+  "private": true,
   "scripts": {
+    "compile": "faucet --compact",
     "start": "npm run compile -- --watch",
-    "compile": "faucet --compact"
+    "preinstall": "npx ensure-node-env"
```
[the remark this answers is not in the document]
Comment (mvitz) [reply package.json]: I've removed the check for the node version.

[the remark this answers is not in the document]
Comment (joyheron) [reply package.json]: Thank you :)

**Comment by joyheron** [comment]
I've tested this locally, and it looks good to me! 

I did get two extra dependencies when I updated the transitive node dependencies (`rm -rf node_modules/ package-lock.json && npm install`), but they may have just snuck in there since yesterday.

As far as I am concerned, this is ready to be merged! :)

**Pull request merged**